corehq/apps/cachehq/cachemodels.py

```diff
@@ -76,7 +76,6 @@ class LocationGenerationCache(GenerationCache):
     generation_key = '#gen#location#'
     doc_types = ['Location']
     views = [
-        'commtrack/locations_by_code',
         '_all_docs',
```

Comment:
yeah i'm guessing I added this so i could cache the contents of `iter_docs` using locations. if that doesn't seem like a thing that needs to happen anymore then fine to remove